cluster/config-defaults.yaml

```diff
@@ -44,15 +44,15 @@ kube_aws_ingress_default_lb_type: "application"
 #
 # "exec": same as "pre" and kube-ingress-aws-controller will default to NLB
 # after removing it you need to set kube_aws_ingress_default_lb_type: "network" above and cleanup skipper and ingress-ctl deployment.yaml
-{{if eq .Cluster.Channel "dev"}}
-nlb_switch: "pre"
-{{else if eq .Cluster.Channel "alpha"}}
-nlb_switch: "false"
-{{else if eq .Cluster.Channel "beta"}}
-nlb_switch: "false"
-{{else}}
-nlb_switch: "false"
-{{end}}
+#
+# new clusters should use NLB as default, everything else has already
+# set config items via cluster registry
+nlb_switch: "exec"
+# TODO(sszuecs): grace period need also to be set for NLB and in skipper deployment we should clean i up
+# skipper termination settings
+skipper_wait_for_healthcheck_interval: "65s"
+# skipper_termination_grace_period: 302
+skipper_termination_grace_period: "70"
```

Comment:
Lets maybe not move these properties around but just add a comment to the existing defaults (since they are not changed)
```yaml
# skipper termination settings
# When nlb_switch is "exec" skipper_termination_grace_period value is ignored and termination grace period is explicitly set to 302 seconds
skipper_termination_grace_period: "70"
skipper_wait_for_healthcheck_interval: "65s"
```

Reply:
why does it matter?

Reply:
* Moving without change makes it harder to track when/why current value was introduced/changed
* commented out `# skipper_termination_grace_period: 302` creates confusion IMO
* I am struggling to understand `TODO(sszuecs): ` comment clearly